<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportsPage</class>
 <widget class="QWidget" name="ReportsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reports</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #2D2D2D; color: #E0E0E0;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="styleSheet">
      <string notr="true">font-size: 24px; font-weight: bold; color: #0078D7;</string>
     </property>
     <property name="text">
      <string>Hospital Reports</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="statsGroupBox">
     <property name="title">
      <string>Statistics</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="totalPatientsLabel">
        <property name="text">
         <string>Total Patients:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="totalPatientsValue">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="totalRevenueLabel">
        <property name="text">
         <string>Total Revenue:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="totalRevenueValue">
        <property name="text">
         <string>$0.00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="doctorsGroupBox">
     <property name="title">
      <string>Most Booked Doctors</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="doctorListWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="generateButton">
       <property name="text">
        <string>Generate Reports</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportCSVButton">
       <property name="text">
        <string>Export to CSV</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportTXTButton">
       <property name="text">
        <string>Export to TXT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>Back to Dashboard</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
